paasta_tools/paasta_remote_run.py

```diff
@@ -230,6 +232,15 @@ def paasta_to_task_config_kwargs(
     return kwargs
 
 
+def task_config_to_dict(task_config):
```

Comment:
Maybe this could be the `__repr__` of a TASK_CONFIG_INTERFACE or something?

Reply:
I tried that but because `task_config` is a PRecord, it doesn't give me a dict that I can pretty print.